node/clients/LMClient.ts

```diff
@@ -24,6 +25,24 @@ export default class LMClient extends ExternalClient {
     })
   }
 
+  public checkUserAdminPermission = async (
+    account: string,
+    userEmail: string,
+    resourceCode: string
+  ) => {
+    const productCode = '97'
```

Comment:
lets make this a constant in the file and add a comment with the resource name, something like this:
`const B2B_LM_PRODUCT_CODE = '97' // resource name on lincense manager = B2B`

Reply:
niceee, much better